### ISSUE TITLE
[BUG] Spectator V5 returning 404 even though summoner is playing arena

### ISSUE THREAD

**Issue by SpectateTheBaus** [RiotGames/developer-relations#923]
This happens from time to time with arena games.
I don't know how to replicate it but it just happened for these games: EUW1_6931217657 and EUW1_6931259007

**Comment by tharuss** [comment]
hello we are riot games and want to tell you that we dont know how to solve this issue, greetings from riot developer team, have a nice day

**Comment by SpectateTheBaus** [comment]
It's happening right now for Thebausffs #EUW it responded with 200 for a minute and now it's back to responding with 404 even though the game is still going.

**Comment by SpectateTheBaus** [comment]
Based on the games that have responded with 200 and the ones that never respond with 200, one difference i've noticed is that that the ones that never responded with 200 all have "queueId": 1710 and the ones that respond with 200 have "queueId": 1700.

**Comment by DarkIntaqt** [comment]
games with the queueId 1710 are custom games and therefore probably cannot be spectated. That's just a guess tho

**Comment by geozukunft** [comment]
With queueId 1710 my guess would be that they are considered somewhat custom lobbies and for some weird reasons spectator not allowing that to be viewed as is the case with QueueID 0

**Comment by SpectateTheBaus** [comment]
That's certainly possible but usually custom games can be spectated unless you choose to disallow spectators at creation.
Also these games show up on match-v5 and sites e.g https://www.leagueofgraphs.com/match/euw/6931259007 unlike queueId 0 custom games.

**Comment by SpectateTheBaus** [comment]
There seems to be another bug when Spectator V5 returns 200 for only a short amount of time (less than 30 seconds) and 404 rest of the game. It happened for this game EUW1_6940196576 but only for Thebausffs#EUW and kermut#000, it returned 200 for every other player in that game.

**Comment by tisbells** [comment]
Hey there - This is expected to be impacted by the same rules as custom lobbies that are able to be spectated (see https://github.com/RiotGames/developer-relations/issues/884). You should still be able to see this queue ID in the spectator API as long as someone has pressed the spectate button in the client. If you find that isn't the case, please let us know.

@SpectateTheBaus are you still having the issue you mentioned of only being able to spectate temporarily?

**Comment by SpectateTheBaus** [comment]
@tisbells Yes it just happened for matchId EUW1_6948387907 (Thebausffs#EUW), it might be because of a player starting a new arena game before the one that they're playing has ended.

**Comment by SpectateTheBaus** [comment]
I checked 4 random arena games and 4 random tft games and there was at least 1 player that responded with 404 even though the rest respond with 200 in 2/4 games for arena and for tft it was 3/4 games.

**Comment by softwareVirus** [comment]
The issue still continue after 1 year.
